### PR TITLE
zh: update full export and import for more columns in downstream

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

support more columns in downstream for full export and import, because we have supported it in https://github.com/pingcap/dm/pull/906.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GMHDBJD @ireneontheway PTAL

**Review by ireneontheway — APPROVED** [review]
LGTM

**Review by GMHDBJD — COMMENTED** [review]
LGTM

File: zh/usage-scenario-downstream-more-columns.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ summary: 了解如何在下游表结构比上游存在更多列的情况下，
 >
 > - 下游相比于上游多的列必须指定默认值或允许为 `NULL`。
 > - 对于已经通过 DM 在正常迁移的表，可直接在下游添加指定了默认值或允许为 `NULL` 的列而不会影响数据迁移。
-> - 当前 DM 暂不支持全量导入表结构不一致的数据到 TiDB 中。
+> - 若下游相比于上游多的列存在默认值或允许为 `NULL`，则对于全量导出与导入无需要特殊配置与处理。
```
Comment (yikeke) [inline zh/usage-scenario-downstream-more-columns.md:14]:
```suggestion
> - 若下游相比于上游多的列存在默认值或允许为 `NULL`，则对于全量导出与导入无需特殊配置与处理。
```

File: zh/usage-scenario-downstream-more-columns.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ summary: 了解如何在下游表结构比上游存在更多列的情况下，
 >
 > - 下游相比于上游多的列必须指定默认值或允许为 `NULL`。
 > - 对于已经通过 DM 在正常迁移的表，可直接在下游添加指定了默认值或允许为 `NULL` 的列而不会影响数据迁移。
-> - 当前 DM 暂不支持全量导入表结构不一致的数据到 TiDB 中。
+> - 若下游相比于上游多的列存在默认值或允许为 `NULL`，则对于全量导出与导入无需特殊配置与处理。
```
Comment (yikeke) [inline zh/usage-scenario-downstream-more-columns.md:14]:
```suggestion
> - 若下游相比于上游多出的列存在默认值或允许为 `NULL`，则对于全量导出与导入无需特殊配置与处理。
```

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**